### ISSUE TITLE
Alternative web base for the app

### ISSUE THREAD

**Issue by GEkuL** [johannesjo/super-productivity-android#23]
Hi there,

I've just installed docker version of the web app, and got the question: how to make the android app connect to exactly this (my server) instead of the default one by official domain?

**Comment by lrq3000** [comment]
You need to make your own build and replace the url pointing to https://app.super-productivity.com/ to point to your own domain instead. Just make sure your domain can serve https content, because Android is now pretty picky about which domains apps can access, and it prevents cleartext communications unless you go through hoops to modify the app's code accordingly.

**Comment by GEkuL** [comment]
@lrq3000 yeah, I wish I could do that via settings, and not just rebuilding the app :(

**Comment by johannesjo** [comment]
@GEkuL I am open for adding this as a feature, but probably won't work on it myself.

**Comment by GEkuL** [comment]
@johannesjo I see, but I'm not into android dev either :(

**Comment by jiongxuan** [comment]
@GEkuL @johannesjo @lrq3000  See: https://github.com/johannesjo/super-productivity-android/issues/50

This problem has been solved. Please refer to README for configuration after updating.https://github.com/johannesjo/super-productivity-android?tab=readme-ov-file#building-locally

Regarding a fully offline solution, I agree that the changes would be significant, and it’s something that needs careful consideration. 

However, I am writing the logic for this and it should be online soon. Please stay tuned.
